protected/humhub/modules/file/handler/AudioFileHandler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\file\handler;
+
+use Yii;
+
+/**
+ * AudioFileHandler allows uploading audio files type
+ * @since 1.15
+ */
+class AudioFileHandler extends BaseFileHandler
+{
+
+    /**
+     * @inheritdoc
+     */
+    public function getLinkAttributes()
+    {
+        return [
+            'label' => Yii::t('MailModule.base', 'Attach an audio message'),
```

Comment:
Not sure that this should be translated via external module (i.e. Mail module), should keep the core and external modules as separated as possible.